src/db/dbConfig.ts

```diff
@@ -0,0 +1,10 @@
+import { Client } from "pg";
+import "dotenv/config";
+
+export const client = new Client({
```

Comment:
this should be handled as a `singelton` like` ExampleService` and `ExampleStore`